synapse/lib/stormlib/model.py

```diff
@@ -812,3 +814,189 @@ async def _methCopyTags(self, src, dst, overwrite=False):
                 for propname, valu in tagprops.items():
                     if overwrite or not dst.hasTagProp(tagname, propname):
                         await proto.setTagProp(tagname, propname, valu) # use tag perms
+
+@s_stormtypes.registry.registerLib
+class LibModelMigrations(s_stormtypes.Lib):
+    '''
+    A Storm library for selectively migrating nodes in the current view.
+    '''
+    _storm_locals = (
+        {'name': 'itSecCpeFixup',
```

Comment:
We may want to use something besides "fixup" here so avoid future collisions.  IMO these function names can be a bit more verbose/specific.